src/modules/commander/HealthAndArmingChecks/checks/cpuResourceCheck.cpp

```diff
@@ -41,6 +41,8 @@ void CpuResourceChecks::checkAndReport(const Context &context, Report &reporter)
 		return;
 	}
 
+	static hrt_abstime high_cpu_load_start_time_us_ = 0;
```

Comment:
Yes this should be a class member now that we can do that.

Reply:
done